optaweb-tsp-planner/pom.xml

```diff
@@ -52,6 +52,11 @@
       <artifactId>spring-boot-starter-test</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.graphhopper</groupId>
+      <artifactId>graphhopper-reader-osm</artifactId>
+      <version>0.11.0</version>
```

Comment:
Why not introducing dependencyManagement already?

Reply:
It looks like `<dependencyManagement>` only makes sense in a parent POM to manage childrens' dependencies: https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Dependency_Management.